concent_api/core/tests/test_core_views.py

```diff
@@ -295,11 +305,11 @@ def test_send_reject_message_save_as_receive_out_of_band_status(self):
             reverse('core:send'),
             data = dump(
                 self.reject_report_computed_task,
-                PROVIDER_PRIVATE_KEY,
+                REQUESTOR_PRIVATE_KEY,
```

Comment:
Why is the key changing here?

Reply:
Because it should be send as requestor. In previous step of this test provider is sending ForceReportComputedTask, it is requestor who must send RejectReportComputedTask.

Reply:
So it was a bug in the test? Please put bugfixes like this in separate commits. It's independent from the feature so it could have gone to master even before we finished the review here.

Reply:
Not really a bug, because with previous implementation it was ok to send it as anyone, now it is not, so it is related with current changes.